site/en_AU.all.json

```diff
@@ -1412,5 +1412,20 @@
   },
   "shopping_error_invalid_session_token": {
     "other": "Shopping session expired, please try again."
+  },
+  "live_lozenge_ended": {
+    "other": "LIVE: ENDED"
+  },
+  "live_lozenge_live": {
+    "other": "LIVE"
+  },
+  "live_lozenge_minutes": {
+    "other": "LIVE: {{.Minutes}} MINUTES"
+  },
+  "live_lozenge_hours": {
+    "other": "LIVE: {{.Hours}} HOURS"
+  },
+  "live_lozenge_days": {
+    "other": "LIVE: {{.Days}} DAYS"
```

Comment:
Just want to check we're okay with these ALL IN CAPS by default and not just converting them to caps in CSS. We are using `text-transform: uppercase` elsewhere in our stylesheets and I feel we should probably be consistent.

Reply:
@matthewthomsonnz Do you have an opinion on this?

Reply:
We don't do text transform in core-template... when a client wants it changed, overriding CSS in a template is more annoying than changing their translations in the db.

Reply:
Uppercase translations is probably best to give clients more control in the future

IMO design was sacrificed with the strictly no transforms dictate. The issue was non-english clients that didn't want uppercase right? It would be nice to allow text transform for en only, or maybe even just the latin languages